sensor_msgs/Image image_1d
---
string color